mgr/src/server/plugins/volume_rename.cr

```diff
@@ -0,0 +1,36 @@
+require "../datastore/*"
+
+post "/api/v1/pools/:pool_name/volumes/:volume_name/rename" do |env|
+  pool_name = env.params.url["pool_name"]
+  volume_name = env.params.url["volume_name"]
+
+  new_volname = env.params.json["new_volname"].as(String)
+
+  next forbidden(env) unless Datastore.maintainer?(env.user_id, pool_name, volume_name)
+
+  pool = Datastore.get_pool(pool_name)
```

Comment:
skip this. Use `volume.pool` when needed.

Reply:
skipped at end, while returning.